src/pixel/mod.rs

```diff
@@ -10,6 +10,15 @@ pub use self::gamma_rgb::GammaRgb;
 mod srgb;
 mod gamma_rgb;
 
+///A transfer function to and from linear space.
+pub trait TransferFn {
```

Comment:
We need a trait to convert gamma encoded colors to linear colors (and the inverse). 

```let rgb = linear_rgb.from_linear()``` currently does not work. If we do not want to add another trait, this is where these conversions will go for every type that can convert to and from linear.

Or we will need something similar to the From and Into traits that will leverage the transfer function to do the linear <-> non-linear conversions.